clippy_lints/src/utils/hir_utils.rs

```diff
@@ -669,7 +669,7 @@ impl<'a, 'tcx> SpanlessHash<'a, 'tcx> {
             TyKind::Typeof(anon_const) => {
                 self.hash_expr(&self.cx.tcx.hir().body(anon_const.body).value);
             },
-            TyKind::CVarArgs(lifetime) => self.hash_lifetime(lifetime),
+            //            TyKind::CVarArgs(lifetime) => self.hash_lifetime(lifetime),
```

Comment:
I think this was already fixed on the master branch :thinking: Maybe you have to rebase on the current Clippy master